ChangeLog.md

```diff
@@ -72,6 +75,11 @@
 
 - Removes the `Pure` effect. It’s unlikely that this will require changes, as `Pure` had no operations, but `Lift Identity` should be used instead. ([#307](https://github.com/fused-effects/fused-effects/pull/307))
 
+- Redefines the `Lift` effect, allowing inner contexts to run actions in outer contexts, e.g. to interoperate with `Control.Exception`. ([#306](https://github.com/fused-effects/fused-effects/pull/306))
+
+- Removes `MonadUnliftIO` instances as they’ve been subsumed by the new definition of `Lift`. ([#306](https://github.com/fused-effects/fused-effects/pull/306))
```

Comment:
We might want to mention that legacy code that depended on a MonadUnliftIO instance could profitably switch to one of the transformers provided by `transformers`.